packages/ses/src/intrinsics.js

```diff
@@ -98,25 +98,36 @@ export const makeIntrinsicsCollector = () => {
       if (typeof permit !== 'object') {
         throw TypeError(`Expected permit object at whitelist.${name}`);
       }
-      const namePrototype = permit.prototype;
-      if (!namePrototype) {
+      const permitPrototype = permit.prototype;
+
+      // Bypass Hermes bug, fixed in: https://github.com/facebook/hermes/commit/00f18c89c720e1c34592bb85a1a8d311e6e99599
```

Comment:
Please explain the Hermes bug you're bypassing below, in addition to the commit link.

Reply:
TODO: add explanation pending investigation how `lockdown` / `harden` / `%InitialGetStackString%` are defined in SES
- if object literal concise method, commit link is correct
- if concise function, https://github.com/facebook/hermes/commit/c42491de94aff479e5e83c073eff96a6261da080

Reply:
just clarifying the above (how these three are defined in SES)

- lockdown: arrow fn
  - harden: arrow fn
- getStackString: object literal concise (shorthand) method

https://github.com/endojs/endo/blob/4ca4028bb67ba9198060f30a7e5bbff21e8fa27f/packages/ses/src/lockdown-shim.js#L15-L18

https://github.com/endojs/endo/blob/4ca4028bb67ba9198060f30a7e5bbff21e8fa27f/packages/ses/src/lockdown.js#L408

https://github.com/endojs/endo/blob/4ca4028bb67ba9198060f30a7e5bbff21e8fa27f/packages/ses/src/error/tame-error-constructor.js#L21-L23

so added commit earlier updating documentation in permits.js and intrinsics.js https://github.com/endojs/endo/pull/2563/commits/575cd60c5d1322604430b89681d37989e2f34239